gitpod-io/gitpod
# Update VS Code Browser to `1.76.2`

## Description
Update code to https://github.com/microsoft/vscode/releases/tag/1.76.2

A part of https://github.com/gitpod-io/gitpod/issues/16587

## Progress


## How to test

- Switch to VS Code Browser Insiders in settings.
- Start a workspace.
- Test the following:
  - that user data is synced across workspaces as well as on workspace restarts, especially for extensions

## Release Notes

```release-note
NONE
```

## Werft options:
